.github/workflows/rosa-scaling-benchmark.yml

```diff
@@ -346,25 +306,30 @@ jobs:
         with:
           createReportFile: false
 
+      - name: Generate timestamp for archive name
+        run: |
+          cur_epoch_time=$(date +%s)
+          echo "ARCHIVE_EPOCH=$cur_epoch_time" >> $GITHUB_ENV
```

Comment:
Would it makes sense to add some name parameter to the workflow name? Can be harder now to distinguish between non-persistent-sessions and persistent-session when there is only a different timestamp. WDYT?

Reply:
That was a question from Anna as well, but it would mean introducing a variable which we have to remove once we switch to persistent sessions only based runs. If it makes it easier, I can work on that change. lmk @mhajas

Reply:
I think it would be cleaner to have the name there. So for me it is +1 to change this but I will leave it to you

Reply:
I will fix this in a subsequent PR along with any trailing issues.